github/assessments.md

```diff
@@ -71,8 +71,12 @@ An assessment is not complete until the assessment is reviewed by an instructor,
 **You have successfully submitted your assessment!**
 
 #### Next Steps
-- An instructor will review your code and leave you comments
-- Open your pull request and click the green `Merge pull request button`
+- An instructor will review your code and leave you comments. 
+  - Possible Instructor responses are:
```

Comment:
I think line 74 and 75 can just be two sentences without bullet points. Then the bullets can start with the list items of approval, etc. Can you lowercase instructor on line 75 please.